src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+
+        int [] arr1 = {1,2,3,4,5,6,7,8,9,10};
```

Comment:
You don't need to initialize an array. Just create it empty with needed size.
`int array[] = new int[size];`